pkg/util/cache/cache_test.go

```diff
@@ -63,3 +66,31 @@ func TestOverwrite(t *testing.T) {
 	cache.Add(1, "yyy")
 	ExpectEntry(t, cache, 1, "yyy")
 }
+
+func TestTooSmall(t *testing.T) {
+	cache := NewCache(shardsCount - 1)
+	if cache != nil {
+		t.Errorf("Expect to cache to be nil, maxSize too small")
+	}
+}
+
+// TestEvict this test will fail sporatically depending on what add()
+// selects for the randomKey to be evicted.  Ensure that randomKey
+// is never the key we most recently added.  Through experimentation,
+// 5 times through the loop is enough to create the issue if it exists.
```

Comment:
5 times should give you 1/32 false pass rate, let's make it 7, which should drop the false pass rate below 1%.